packages/frame-core/src/types.ts

```diff
@@ -18,7 +18,15 @@ export type AccountLocation = {
   description: string;
 };
 
-export type FrameLocationNotificationContext = {
+export type FrameLocationContextCastEmbed = {
+  type: "cast_embed";
+  cast: {
+    fid: number;
+    hash: string;
```

Comment:
starting conservative, no `text` due to then also having to have mentions, mentionsPositions, etc